staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -526,6 +526,14 @@ func (f *DeltaFIFO) ListKeys() []string {
 	return list
 }
 
+// Count returns a count of all the keys of the objects currently
+// in the FIFO.
```

Comment:
hmm - the semantics of Count() for DeltaFIFO/FIFO is somewhat questionable to me.

With the recent refactor, where watchCache has its own `storeIndexer` interface defined, we call `NewIndexer` there, I don't think we need it here.

I think we only need to introduce `Count` method in `Indexer` interface and then add it only to:
ThreadSafeStore
threadSafeMap
cache

Can you please verify that?

Reply:
I have changed to add Count method in Indexer interface, PTAL.